src/forms/FilterForm/FilterForm.js

```diff
@@ -58,6 +58,25 @@ const FilterFormComponent = props => {
 
         const classes = classNames(css.root);
 
+        const spy =
+          liveEdit || onChange ? (
+            <FormSpy onChange={handleChange} subscription={{ values: true, dirty: true }} />
```

Comment:
My experience with these FormSpy setups is that they are extremely tricky. Check couple of times (even though this is an existing setup).

Reply:
Not sure what I should've looked for, but haven't noticed any problems this far.